backend/database/migrations/create-users.js

```diff
@@ -24,6 +24,9 @@ module.exports = {
       updatedAt: {
         allowNull: false,
         type: Sequelize.DATE
+      },
+      authToken: {
+        type: Sequelize.STRING
```

Comment:
We don't need to add an authToken field, the token would be a [JWT](https://jwt.io/) which would contain encrypted user info, so decoding it would output the user